game/scripts/npc/items/custom/item_cave_rune_haste_doubledamage.txt

```diff
@@ -0,0 +1,38 @@
+"DOTAItems"
+{
+//============================================================================================================
+//General
+//============================================================================================================
+"item_cave_rune_haste_doubledamage"
+{
+	//General
+	//----------------------------------------------------------------------------------------------------
+	"ID" 				"9999" PLEASE GIVE THIS A REAL NUMBER
+	"BaseClass" 			"item_lua"
+	"AbilityTextureName"		"item_cave_rune_haste_doubledamage"
+	"ScriptFile" 	 		"items/cave_rune_hasted_doubledamage.lua"
+
+	"AbilityBehavior" 		"DOTA_ABILITY_BEHAVIOR_IMMEDIATE | DOTA_ABILITY_BEHAVIOR_NO_TARGET"
+
+	"Model"				"models/props_gameplay/bottle_empty.vmdl"
+
+	//Stats
+	//----------------------------------------------------------------------------------------------------
+	"AbilityCooldown"		"0.5"	
+	"AbilityCastPoint"		"0.0"
+
+	//Item Info
+	//----------------------------------------------------------------------------------------------------
+	"ItemCost" 			"1300"
+	"ItemStackable" 		"1"

```

Comment:
Are these really supposed to be stackable?

Reply:
Unsure, but I thought about how Baumi does things, and it seemed like a valid strategy to clear the cave twice and double the buff duration in the next duel. Plus, it means that only one person on the team has to carry the bottles, and can just hand them out before the duel. So, with that in mind, I thought stacking would enrich gameplay.

Plus, Baumi always says to make something OP initially and then nerf it as needed.

Reply:
Stackable in this context just means the duration extents right? In that case, yes it should be stackable.

Reply:
It means that multiples of a single bottle will stack like tangos. Or infinity bottles.

Reply:
Oh. Then no.

Reply:
It can be changed to consume all stacks in a single use and extend duration, if that would be desirable.

Reply:
Nono, that's unintuitive. Just unstackable is fine. They are not really bottles, they are single use consumables. Think of them like a Mango.